### PR TITLE
Compute v2: Create aggregate

### PR DESCRIPTION
**Pull request by dstdfx**
For #738 

Links to the line numbers/files in the OpenStack source code that support the
code in this PR:

API doc: 
- https://developer.openstack.org/api-ref/compute/#create-aggregate 

Source code:
- https://github.com/openstack/nova/blob/master/nova/api/openstack/compute/aggregates.py#L56

### CODE REVIEW

**Comment by dstdfx** [comment]
@jtopjian I'll add acceptance tests for this PR and #740 after they are merged. It must be useful to test them together in a single testcase.

**Comment by dstdfx** [comment]
recheck

File: openstack/compute/v2/extensions/aggregates/requests.go [hunk 1]
```diff
@@ -11,3 +11,31 @@ func List(client *gophercloud.ServiceClient) pagination.Pager {
 		return AggregatesPage{pagination.SinglePageBase(r)}
 	})
 }
+
+type CreateOpts struct {
+	// The name of the host aggregate.
+	Name string `json:"name"`
```
Comment (jtopjian) [inline openstack/compute/v2/extensions/aggregates/requests.go]: `name` is a required field, so this should be:

```go
Name string `json:"name" required:"true"`
```

File: openstack/compute/v2/extensions/aggregates/doc.go [hunk 2]
```diff
@@ -1,7 +1,21 @@
 /*
-Package aggregates returns information about the host aggregates in the
+Package aggregates creates aggregates and returns information about the host aggregates in the
```
Comment (jtopjian) [inline openstack/compute/v2/extensions/aggregates/doc.go]: Might as well rename this to:

```
Package aggregates manages information about the host aggregates in the
OpenStack cloud
```

**Comment by dstdfx** [comment]
@jtopjian Okay, will keep it in mind.

**Comment by jtopjian** [comment]
@dstdfx Looks good to me. Thank you for working on this. :)

**Pull request merged**